### ISSUE TITLE
Haxe Libraries to Produce Single Merged SWC

### ISSUE THREAD

**Issue by rat-moonshine** [Moonshine-IDE/Moonshine-IDE#875]
-- email subject "Style declaration problems from different Haxe libraries in Moonshine" --

**Comment by rat-moonshine** [comment]
@joshtynjala I'm seeing source-problem while working on Code:

![image](https://user-images.githubusercontent.com/24370870/124260675-94c78300-db4d-11eb-97a5-1ae799bbdbc0.png)

It looks like Code failed to read the `MoonshineGUICore` configuration properly:
```
<source path="src" />
<source path="externs" />
<source path="../../../Moonshine-SDK-Installer/MoonshineSDKInstallerGUICore/src" />
<source path="../../../Moonshine-SDK-Installer/MoonshineSDKInstallerGUICore/externs" />
```
However, declaring the same way Code has no problem accessing `SharedHaxe` source when working in `GBAuthHaxe`.

**Comment by rat-moonshine** [comment]
I have completed first batch of changes doing `MoonshineGUICore` style-sheet `extends` `MoonshineSDKInstallerGUICore`, and I already starts seeing fix of previously broken stuff, including the Haxe popup background and layout issue.

![image](https://user-images.githubusercontent.com/24370870/124275654-76b74e00-db60-11eb-8f4b-50257d2bfc26.png)

**Comment by joshtynjala** [comment]
> Do you have an insight how to fix that? I don't have problem in compilation, though.

Try restarting VSCode. If that doesn't work, delete the bin directory and then, try restarting again.

**Comment by rat-moonshine** [comment]
This issue is now merged onto `master`.

@joshtynjala @piotrzarzycki21 @feather812002 @JoelProminic Project's [configuration](https://github.com/Moonshine-IDE/Moonshine-IDE/blob/master/ide/MoonshineDESKTOPevolved/MoonshineDESKTOPevolved.as3proj) file is now updated with the requisite changes. In case you're working with different IDE using different configuration file, please make sure to remove dependency for `MoonshineSDKInstallerGUICore.swc` from its settings as the library shall produce as a merged-SWC as `MoonshineGUICore.swc`.

**Comment by JoelProminic** [comment]
I noticed that the "Only show features that need setup" checkbox does not work with the updated build.  @rat-moonshine thought it was probably a case that was missed for this issue, but we can move it to a separate issue if appropriate:

![image](https://user-images.githubusercontent.com/24532731/124643694-fe4dd700-de56-11eb-837e-bc5f72e7d1ae.png)

**Comment by rat-moonshine** [comment]
@JoelProminic is the problem you pointed towards to no filtered-item(s) displaying? 

I was tested with build.2024, and I see the filtration worked well as I had only item that not being installed yet:
![image](https://user-images.githubusercontent.com/24370870/124717866-286ccb00-df23-11eb-87ff-b9d2adfeaa98.png)

If the filtration do not shows you the item(s) that might needs-to-be-installed, which are those items?

**Comment by JoelProminic** [comment]
Yeah, this was because all of the SDKs were configured.   I feel like we should have a default message here when there are no matches, but this should be moved to a separate issue.

Otherwise, the updates looks good.

**Comment by piotrzarzycki21** [comment]
Please close this issue.

**Comment by JoelProminic** [comment]
I extracted my last comment to #876.  If this is an easy fix, then we can include it in 3.2.0.  Otherwise, it is a low priority.
